tests/integration/suite/conftest.py

```diff
@@ -109,6 +109,13 @@ def admin_mc():
     return ManagementContext(client, k8s_client, user=admin)
 
 
+@pytest.fixture
+def admin_proxy(admin_mc):
```

Comment:
It looks like this is just adding BASE_URL, if so just add it to the normal admin fixture but change the field name to be snake case